tests/src/equipNft.test.ts

```diff
@@ -119,16 +119,34 @@ async function checkEquipStatus(
   api: ApiPromise,
   expectedStatus: "equipped" | "unequipped",
   collectionId: number,
-  nftId: number
+  nftId: number,
+  expectedResourceId?: number,
+  expectedSlotId?: number
 ) {
   const itemNftDataOpt = await getNft(api, collectionId, nftId);
   expect(itemNftDataOpt.isSome, "Error: unable to fetch item NFT data");
 
   const itemNftData = itemNftDataOpt.unwrap();
   expect(
-    itemNftData.equipped.isTrue,
+    itemNftData.equipped.isSome,
     `Error: item NFT should be ${expectedStatus}`
   ).to.be.equal(expectedStatus === "equipped");
+
+  if (
+    expectedStatus === "equipped" &&
+    expectedResourceId !== undefined &&
+    expectedSlotId !== undefined
```

Comment:
minor thing, `!== undefined` is kinda redundant so a bit shorter and concise just to write as:
```
  if (expectedStatus === "equipped" && expectedResourceId && expectedSlotId)
```

Reply:
Yes, you are right, I updated this.